mmdet3d/models/dense_heads/point_rpn_head.py

```diff
@@ -169,18 +179,25 @@ def loss(self,
 
         return losses
 
-    def get_targets(self, points, gt_bboxes_3d, gt_labels_3d):
+    def get_targets(self, points: List[Tensor],
+                    batch_gt_instances_3d: InstanceList) -> Tuple[Tensor]:
         """Generate targets of PointRCNN RPN head.
 
         Args:
             points (list[torch.Tensor]): Points of each batch.
-            gt_bboxes_3d (list[:obj:`BaseInstance3DBoxes`]): Ground truth
-                bboxes of each batch.
-            gt_labels_3d (list[torch.Tensor]): Labels of each batch.
+            batch_gt_instances_3d (list[:obj:`InstanceData`]): Batch of
+                gt_instances. It usually includes ``bboxes_3d`` and
+                ``labels_3d`` attributes.
 
         Returns:
             tuple[torch.Tensor]: Targets of PointRCNN RPN head.
         """
+        gt_labels_3d = [
+            instances.labels_3d for instances in batch_gt_instances_3d
+        ]
+        gt_bboxes_3d = [
+            instances.bboxes_3d for instances in batch_gt_instances_3d
+        ]
         # find empty example
         for index in range(len(gt_labels_3d)):
```

Comment:
Please check whether the code needs to be kept, since after refactor, typically we won't use point cloud without 3D bboxes for training